variables.tf

```diff
@@ -37,6 +37,12 @@ variable "lambda_package_key" {
   default     = null
 }
 
+variable "ephemeral_storage_size" {
+  description = "The size of the Lambda function Ephemeral storage(/tmp) represented in MB. The minimum supported ephemeral_storage value defaults to 512MB and the maximum supported value is 10240MB."
+  type        = number
+  default     = 512
```

Comment:
Would this benefit from [validation](https://developer.hashicorp.com/terraform/language/values/variables#custom-validation-rules) at the module level or will invalid values result in a plan failure?